js/components/urlBar.js

```diff
@@ -176,9 +177,20 @@ class UrlBar extends ImmutableComponent {
             // load the selected suggestion
             this.urlBarSuggestions.clickSelected(e)
           } else {
+            if (this.activateSearchEngine && this.searchSelectEntry !== null && !isLocationUrl) {
+              const replaceRE = new RegExp('^' + this.searchSelectEntry.shortcut + ' ', 'g')
```

Comment:
@darkdh that would be great actually! @aekeus, would you be able to pull this into the [formatUtil](https://github.com/brave/browser-laptop/blob/master/app/common/lib/formatUtil.js) module?